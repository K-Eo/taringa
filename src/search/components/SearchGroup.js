import React from 'react'

const SearchGroup = ({ className, ...props }) => {
  return (
    <div className={className}>
      <h6 className="text-secondary">{props.title}</h6>

      {props.wrapper && <div className="row">{props.children}</div>}

      {!props.wrapper && props.children}
    </div>
  )
}

export default SearchGroup
