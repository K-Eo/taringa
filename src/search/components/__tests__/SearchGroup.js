import React from 'react'

import SearchGroup from '../SearchGroup'

describe('Serach Group', () => {
  it('exists', () => {
    expect(SearchGroup).toBeDefined()
  })

  it('renders', () => {
    const tree = renderer
      .create(
        <SearchGroup title="foo" className="bar">
          baz
        </SearchGroup>
      ).toJSON()

    expect(tree).toMatchSnapshot()
  })

  describe('wrapper', () => {
    it('renders', () => {
      const tree = renderer
        .create(
          <SearchGroup title="foo" className="bar" wrapper>
            baz
          </SearchGroup>
        ).toJSON()

      expect(tree).toMatchSnapshot()
    })
  })
})
