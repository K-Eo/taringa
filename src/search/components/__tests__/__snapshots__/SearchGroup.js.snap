// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Serach Group renders 1`] = `
<div
  className="bar"
>
  <h6
    className="text-secondary"
  >
    foo
  </h6>
  baz
</div>
`;

exports[`Serach Group wrapper renders 1`] = `
<div
  className="bar"
>
  <h6
    className="text-secondary"
  >
    foo
  </h6>
  <div
    className="row"
  >
    baz
  </div>
</div>
`;
